meta-facebook/yv35-cl/src/ipmi/plat_ipmi.c

```diff
@@ -1080,6 +1081,31 @@ void pal_OEM_ACCURACY_SENSNR(ipmi_msg *msg) {
   return;
 }
 
+void pal_OEM_ASD_INIT(ipmi_msg *msg) {
+  if (!msg){
+    printf("pal_OEM_ASD_INIT: parameter msg is NULL\n");
+    return;
+  }
+
+  if (msg->data_len != 1) {
+    msg->completion_code = CC_INVALID_LENGTH;
+    return;
+  }
+
+  if (msg->data[0] == 0x01) {
```

Comment:
What should the behavior be if the value in msg->data[0] is not 0x01 or 0xff (for instance, it is 0xee)?

Reply:
It will return completion code 0xCC (invalid data field).

Reply:
Now, It won't change current interrupt state.
The GPIO interrupt is disabled in default. Same as default state, we will modify code to disable GPIO interrupt when msg->data[0] is not 0x01 or 0xff.